docs/can-fixture.md

```diff
@@ -9,145 +9,225 @@
 
 @signature `fixture(ajaxSettings, requestHandler(...))`
 
```

Comment:
In the description, we should use a more active tone, be from the user's perspective, and not self reference the package.  Instead of: 

> can-fixture intercepts an AJAX request and simulates the response with a file or function.

more like:

> Intercept AJAX requests and simulate the response.